api/app/src/main/resources/application.properties

```diff
@@ -3,10 +3,22 @@ spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL82Dialec
 spring.jpa.properties.hibernate.format_sql=true
 spring.jpa.open-in-view=false
 spring.task.scheduling.pool.size=5
-
 #security
 spring.security.oauth2.client.registration.github.client-id=${GITHUB_CLIENT_ID}
 spring.security.oauth2.client.registration.github.client-secret=${GITHUB_CLIENT_SECRET}
 spring.security.oauth2.client.registration.github.scope=read:org
 spring.security.oauth2.client.registration.github.redirect-uri=${PACKIT_API_ROOT}/login/oauth2/code/github
+#db
+outpack.server.url=${OUTPACK_SERVER_URL:http://localhost:8000}
+db.url=${DB_URL:jdbc:postgresql://localhost:5432/packit?stringtype=unspecified}
+db.user=${DB_USER:packituser}
+db.password=${DB_PASSWORD:changeme}
+#Auth
+auth.jwt.secret=${JWT_SECRET:changesecretkey}
+auth.oauth2.redirect.url=${AUTH_REDIRECT_URL:http://localhost:3000/redirect/}
+auth.method=${AUTH_METHOD:github}
+auth.expiryDays=${JWT_EXPIRY_DAYS:1}
+auth.enabled=${AUTH_ENABLED:true}
+auth.githubAPIOrg=${AUTH_GITHUB_ORG:mrc-ide}
+auth.githubAPITeam=${AUTH_GITHUB_TEAM:packit}
```

Comment:
I think it would be a good idea to scope all the environment variables with a PACKIT_ prefix - just to show that they're all related and avoid any clashes with other env vars - e.g. "DB_USER" is pretty generic!